docs/reference/builder.md

```diff
@@ -112,9 +112,14 @@ Here is the format of the `Dockerfile`:
 The instruction is not case-sensitive, however convention is for them to
 be UPPERCASE in order to distinguish them from arguments more easily.
 
-Docker runs the instructions in a `Dockerfile` in order. **The
-first instruction must be \`FROM\`** in order to specify the [*Base
-Image*](glossary.md#base-image) from which you are building.
+The instruction can be either a command executed by the build engine, or a
+parser directive affecting the way in which the remainder of the `Dockerfile` is
+interpreted.
+
+Docker runs instructions in a `Dockerfile` in order. **The
+first instruction should be \`FROM\`** in order to specify the [*Base
```

Comment:
why the s/must/should/ ?  Maybe if you change "instruction" to "command" (to align with your wording above) then you can keep the 'must' ?